### PR TITLE
Vomiting no longer makes you fall on the ground

### PR DESCRIPTION
**Pull request by Mickyan**
Back when stuns were refactored everything was rolled into Paralyze which had the unintended effect of making you fall on the ground after vomiting.  As a frequent patron of the bar this always bothered me because it looks stupid and today I am finally doing something about it

This shouldn't have any noteworthy changes to gameplay, you still get stunned and everything you just don't fall on the floor anymore
:cl:
fix: Alcoholics rejoice: the stun from vomiting no longer makes you fall on the floor
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
https://github.com/tgstation/tgstation/pull/39967/files#diff-6eecaffa18a39474a130993b613476d23e014e01a673d33de1ecbfbb80478c96

It knocked you down before, removing the knockdown isn't a fix

**Comment by Mickyan** [comment]
Then maybe it predates the last rework because I am 100% sure it didn't use to but whatever I'll take it

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -778,7 +778,7 @@
 			visible_message(span_warning("[src] dry heaves!"), \
 							span_userdanger("You try to throw up, but there's nothing in your stomach!"))
 		if(stun)
-			Paralyze(200)
+			Stun(200)
```
Comment (san7890) [inline code/modules/mob/living/carbon/human/human.dm:781]:
```suggestion
			Stun(20 SECONDS)
```

defines are better than raw deciseconds

**Comment by ShizCalev** [comment]
image

**Pull request merged**